### PR TITLE
Enable build S.R.WindowRuntime for .netcoreapp

### PR DESCRIPTION
**Pull request by luqunl**
This change is to allow build S.R.WindowsRuntime.dll for .netcoreapp on windows.

### CODE REVIEW

**Comment by ericstj** [comment]
This will enable the build but doesn't include it anywhere.  I would have expected you to either set `< IsNETCoreApp>true</IsNETCoreApp>` in the library dir.props or add a pkgproj.

File: src/mscorlib.WinRT-Facade/ref/Configurations.props [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       uap;
+      netcoreapp-Windows_NT;
```
[the remark this answers is not in the document]
Comment (luqunl) [reply src/mscorlib.WinRT-Facade/ref/Configurations.props]: Removed Windows_NT

[the remark this answers is not in the document]
Comment (ericstj) [reply src/mscorlib.WinRT-Facade/ref/Configurations.props]: same goes for the other ref.

**Comment by luqunl** [comment]
> I would have expected you to either set < IsNETCoreApp>true</IsNETCoreApp> in the library dir.props or add a pkgproj. I guess you can always do that later.
Jose is helping me on this part and We will add these later.

File: src/mscorlib.WinRT-Facade/ref/Configurations.props (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericstj) [reply src/mscorlib.WinRT-Facade/ref/Configurations.props]: You may find this commit gets you a good part of the way to what you're trying to do, though I do disable validation https://github.com/ericstj/corefx/commit/9e5e4db1d50ae7b0d201249a81d0dd8e4d30cabf#diff-565892acea448d6f53b8dc593d51e9e2R5

[the remark this answers is not in the document]
Comment (luqunl) [reply src/mscorlib.WinRT-Facade/ref/Configurations.props]: Your commit is exactly what we try to do. Jose and I will mimic your change about packaging after decoupling these assemblies dependency on S.P.Corelib.

[the remark this answers is not in the document]
Comment (ericstj) [reply src/mscorlib.WinRT-Facade/ref/Configurations.props]: What I didn’t do with that commit:
1) build a not-supported assembly for non-windows
2) support same TFMs as previous package

Both need to be done eventually to get the package passing validation

**Comment by luqunl** [comment]
@dotnet-bot test NETFX x86 Release Build please

**Comment by luqunl** [comment]
> 
2018-05-08 21:41:40,440: ERROR: job(44): kill: Job running for too long. Killing...
2018-05-08 21:41:40,457: ERROR: executor(403): _execute_command: Executor timed out after 1200 seconds and was killed.
2018-05-08 21:41:40,457: INFO: event(43): send: Sending event type WorkItemTimeout
2018-05-08 21:41:40,628: INFO: saferequests(90): request_with_retry: Response complete with status code '201'
2018-05-08 21:41:40,628: INFO: executor(422): _execute_command: Finished _execute_command, exit code: -3

https://mc.dot.net/#/user/luqunl/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/deaa24a97dec600867b301175241503ee41a2f28/workItem/System.Diagnostics.PerformanceCounter.Tests/wilogs

sounds like there is an infra issue which causes "NETFX x86 Release Build" failure

**Comment by luqunl** [comment]
@dotnet-bot  te@dotnet-bot test NETFX x86 Release Build please

**Pull request merged**